src/App.css

```diff
@@ -1,4 +1,6 @@
 @import url("https://unfold-ui.netlify.app/css/components.css");
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
```

Comment:
Add all your styles imports in `index.html` file.

Reply:
> Add all your styles imports in `index.html` file.

Sure I will do that